js/src/hooks/useCroppedImageSelector/useCroppedImageSelector.test.js

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
💅  Thanks for adding tests. It would be nice to cover also the `useCroppedImageSelector` itself and not only the aux functions. But not a blocker and could be done in the future.

Reply:
@puntope, thanks for the suggestion.

However, I don't know if there is a proper way to test `useCroppedImageSelector` with `jest` since it heavily relies on `window.wp.media` library directly and the library depends on `jQuery` and `backbone.js`. In runtime, they are prebuilt files loaded via WP core PHP functions. And mocking the entire `wp.media` seems impractical, like trying to test a React component by mocking the entire `react` library.

Or do you know of a proper way to do so?